### ISSUE TITLE
--insecure-skip-tls-verification flag no longer works

### ISSUE THREAD

**Issue by Joee94** [runelite/runelite#16295]
### Type

Incorrect behavior

### Operating System

MacOS

### OS Version

Monterey 12.5.1

### Bug description

I need to use the `--insecure-skip-tls-verification` flag in order to run RuneLite on this machine from the command line. As of this morning I get this error 
```2023-02-19 09:29:32 [main] ERROR net.runelite.client.RuneLite - Uncaught exception:
java.lang.NullPointerException: null
	at okhttp3.internal.tls.BasicTrustRootIndex.<init>(BasicTrustRootIndex.java:32)
	at okhttp3.internal.platform.Platform.buildTrustRootIndex(Platform.java:303)
	at okhttp3.internal.platform.Platform.buildCertificateChainCleaner(Platform.java:173)
	at okhttp3.internal.tls.CertificateChainCleaner.get(CertificateChainCleaner.java:41)
```

I noticed in the most recent commit `5ff774fed5a8b075f475bf1d6e9ba857b63c847e` that there are some changes around this.

### RuneLite version

Runelite Version: Unknown but I presume 1.9.11 (No mention of it in console logs)
Launcher Version: 2.6.1

### Logs

[client.log](https://github.com/runelite/runelite/files/10776545/client.log)
[launcher.log](https://github.com/runelite/runelite/files/10776546/launcher.log)

**Comment by Adam-** [comment]
fixed in cfaebe145fbcd7c32439870019d05078b69ed6d3

**Comment by Joee94** [comment]
> fixed in [cfaebe1](https://github.com/runelite/runelite/commit/cfaebe145fbcd7c32439870019d05078b69ed6d3)

Hero

**Comment by amp4045** [comment]
Is there an ETA when this patch will be released without requiring we build the client ourselves?
